Drop-In/build.gradle

```diff
@@ -73,7 +73,8 @@ dependencies {
     api 'com.braintreepayments.api:card:4.1.0'
     api 'com.braintreepayments.api:data-collector:4.1.0'
     api 'com.braintreepayments.api:union-pay:4.1.0'
-    api 'com.braintreepayments:card-form:5.1.0'
+
+    api 'com.braintreepayments:card-form:5.1.2-SNAPSHOT'
```

Comment:
Changes to card form are drafted [here](https://github.com/braintree/android-card-form/pull/109), so we are using a local build for testing (CI will fail for now)